extensions/graphql/src/decorators/index.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/graphql
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+// Force require('reflect-metadata');
+import '@loopback/metadata';
+import {
+  Arg,
+  Args,
+  ArgsType,
+  Field,
+  FieldResolver,
+  InputType,
+  Mutation,
+  ObjectType,
+  Query,
+  Resolver,
+  Root,
+} from 'type-graphql';
+
+// export namespace graphql {
+export const arg = Arg;
```

Comment:
Why are we exporting lowercased version?

Reply:
See https://github.com/strongloop/loopback-next/pull/5545#issuecomment-634220374